README.md

```diff
@@ -54,6 +54,7 @@ rules.
 - [Envoy Mobile](https://github.com/envoyproxy/envoy-mobile)
 - Spotify
 - [SwiftLint](https://github.com/realm/SwiftLint)
+- [Robinhood](https://github.com/robinhoodmarkets)
```

Comment:
Can you put this before the `S` eateries? Thanks!

Also, the links are to show public implementations or proof of use (like blog posts or talks), so can you remove the link as well?

Reply:
Yeah makes sense